download-sony-blobs.sh

```diff
@@ -9,7 +9,7 @@ TOMORROW=$((${NOW} + 1000 * 60 * 60 * 24 * 1))
 
 DIR=download-shinano
 
-if [ ! -x "$(which w3m)" ]; then
+if [ ! -x "$(which w3m)" ] && [ ! -f "${DIR}/${ZIP_FILE}"]; then
```

Comment:
We should rather do the check later, just before calling w3m, since sha1sum will already check this